packages/webapp/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
-        "target": "ES5",
-        "lib": ["DOM", "DOM.Iterable", "ESNext"],
+        "target": "ES2021",
+        "lib": ["DOM", "DOM.Iterable", "ES2021"],
```

Comment:
i am curious about those changes?

Reply:
ah yes forgot to put a comment . replaceAll is only available for > ES2020 and this
```ts
type 'IterableIterator<[string, string]>' can only be iterated through when using the '--downlevelIteration' flag or with a '--target' of 'es2015' or higher.

19             [...usp.entries()].forEach(([key, value]) => {
```